src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml

```diff
@@ -150,6 +150,12 @@ mappings:
                     match_mapping_type: 'boolean'
                     mapping:
                         type: 'boolean'
+            -
+                date:
+                    path_match: 'values.*-date.*'
+                    match_mapping_type: 'string'
+                    mapping:
+                        type: 'keyword'
```

Comment:
Also, there is no mapping for updated/created (which is wierd so if you could that one in your PR that would awesome 👍 )